cpp-utils/include/ot_utils/freertos/freertos_timer.hpp

```diff
@@ -6,6 +6,9 @@
 #include "task.h"
 #include "timers.h"
 
+#define ROUNDED_DIV(A, B) (((A) + ((B) / 2)) / (B))
+#define PASSED_MS(TICKS) (uint32_t)ROUNDED_DIV((TICKS) * 1000, configTICK_RATE_HZ)
```

Comment:
can we name this like `TICKS_TO_MS` or something, it's a little unclear just from the name what it does.